src/makeInfinityAdder.js

```diff
@@ -6,7 +6,34 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
-}
+  let sum = 0;
+  const functionA = numberA => {
+    if (numberA === undefined) {
```

Comment:
Replace with `!numberA`, it will be the same as` numberA === undefined`

Rename `numberA` to `a`

Reply:
Same situation with `numberB`